go.mod

```diff
@@ -405,7 +405,7 @@ replace (
 	github.com/docker/go-plugins-helpers => github.com/elastic/go-plugins-helpers v0.0.0-20200207104224-bdf17607b79f
 	github.com/dop251/goja => github.com/andrewkroh/goja v0.0.0-20190128172624-dd2ac4456e20
 	github.com/dop251/goja_nodejs => github.com/dop251/goja_nodejs v0.0.0-20171011081505-adff31b136e6
-
+	github.com/elastic/elastic-agent-autodiscover => /Users/tetianakravchenko/go/src/github.com/elastic/elastic-agent-autodiscover
```

Comment:
TODO: replace with https://github.com/elastic/elastic-agent-autodiscover/pull/72 when released

Reply:
Reminder: We need the same for elastic-agent repo: https://github.com/elastic/elastic-agent/blob/main/go.mod#L15